1.8/administration/access-node/tunnel.md

```diff
@@ -0,0 +1,113 @@
+---
+post_title: Access by Proxy
+menu_order: 10
+---
+
+When developing applications on DC/OS, you may find it helpful to access your cluster via SOCKS proxy, HTTP proxy, or VPN. For instance, you can work from your own development environment and immediately test on your DC/OS cluster.
+
+# SOCKS
+dcos tunnel can run a SOCKS proxy over SSH to the cluster. SOCKS proxies work for any kind of network protocol on any port, but your application must be configured to use the proxy, which runs on port 1080 by default. Access URLs by appending the `mydcos.directory` domain.
```

Comment:
Instead of "SOCKS proxies work for any kind of network protocol on any port" I'd just say "SOCKS proxies work for any protocol". It doesn't deal with the network protocols, a more pedantic version would be "application protocol" I suppose, but I think just plain "protocol" is more easily understood.

And all of the proxies can access all ports so that isn't particular to this one.